website/docs/faq.md

```diff
@@ -1,19 +1,31 @@
 # FAQ
 
-## What packages does release-plz publish?
+## What packages does release-plz release?
 
-Release-plz publishes all packages, except:
+Release-plz can release packages in multiple ways, such as:
+
+- Publishing to a Cargo registry
+- Creating git tags
+- Creating repository releases
+
+By default, release-plz considers all packages in a workspace for releasing.
+The following are not published to a Cargo registry:
```

Comment:
```suggestion
By default, release-plz considers all packages in a workspace for releasing.
The following are not published to a Cargo registry:
```
Sorry but I strongly disagree with this.

If a package has `publish = false` in the `Cargo.toml` we shouldn't have them in the release PR and we shouldn't create a git tag for it UNLESS there's an explicit `git_tag_enable = true` for that crate (i.e. workspaces would require a `[[package]]` override). If this is too boring, we could consider adding a new config flag for this in another PR.

Otherwise this will be a huge breaking change for every existing user because people will start seeing tags for crates that until now where ignored in their workspace.

What do you think?